lib/worker-pipeline.js

```diff
@@ -69,6 +70,29 @@ class AssetComputeWorkerPipeline {
         return engine;
     }
 
+    normalizeInputOuput(input, output) {
+        console.log('input before normalization: ', input);
+        console.log('outpu before normalization: ', output);
+
+        if(output.fmt === 'machine-metadata-json') {
+            output.type = 'machine-metadata-json';
+        } else {
+            // rendition.fmt should always exist
+            const mimetype = mime.lookup(output.fmt.toLowerCase());
```

Comment:
does mime.lookup throw if there is no valid mimetype or in any other case?

Reply:
Doesn't throw, but returns `false`. Added a test for it.
https://www.npmjs.com/package/mime-types -> `All functions return false if input is invalid or not found.`